src/test/java/edu/ucsb/cs156/happiercows/controllers/CourseControllerTests.java

```diff
@@ -0,0 +1,78 @@
+package edu.ucsb.cs156.happiercows.controllers;
+
+import edu.ucsb.cs156.happiercows.ControllerTestCase;
+import edu.ucsb.cs156.happiercows.entities.Course;
+import edu.ucsb.cs156.happiercows.repositories.CourseRepository;
+import edu.ucsb.cs156.happiercows.repositories.UserRepository;
+import edu.ucsb.cs156.happiercows.testconfig.TestConfig;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.orm.jpa.AutoConfigureDataJpa;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureDataJpa
+@WebMvcTest(controllers = CourseController.class)
+@Import(TestConfig.class)
+public class CourseControllerTests extends ControllerTestCase {
+    @MockBean
+    CourseRepository courseRepository;
+
+    @MockBean
+    UserRepository userRepository;
+
+    // @Autowired
+    // private ObjectMapper objectMapper;
```

Comment:
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed

Reply:
yep good catch!